src/foam/u2/view/FObjectView.js

```diff
@@ -52,40 +59,50 @@ foam.CLASS({
       name: 'of'
     },
     {
+      class: 'Array',
       name: 'choices',
```

Comment:
The actual version will show just two type (User and Group). Did this a desired behaviour? or we need to get the exhaustive list of model.

Reply:
We need to add all of the models we want to show up in the list to the `strategyReferences` file. I only added `Group` and `User` as part of this PR, but I can add anything else if it makes sense. Most of our models are on the nanopay side though, so I'd have to add them in a nanopay PR.